README.md

```diff
@@ -1485,7 +1485,13 @@ Make function raise-safe
 
 ### callerFilename(depth = 0, stack = null)
 
-### callerFilepath(depth = 0, stack = null)
+### callerFilepath(\[depth\[, stack\]\])
```

Comment:
Can you keep the default values in the signature?

Reply:
Ok, though metadoc currently makes them `### callerFilename(depth=0, stack=null)` (removes whitespaces and shows warning). For now I added the spaces manually.

/cc @SemenchenkoVitaliy